manifests/apache.pp

```diff
@@ -116,7 +116,7 @@
     }
 
     # This file is installed by pulp-server but we have everything in the above vhost
-    file {'/etc/httpd/conf.d/pulp.conf':
+    file {'/etc/httpd/conf.d/10-pulp.conf':
```

Comment:
This won't work because (as the comment above states) is created by the pulp-server RPM.

Reply:
Ahh right, so we can leave it be as it was.

Reply:
Will https://github.com/theforeman/puppet-pulpcore/pull/50 alone be sufficient?